src/openms/source/FORMAT/IBSpectraFile.cpp

```diff
@@ -213,11 +208,9 @@ namespace OpenMS
     TextFile textFile;
     textFile.addLine(ListUtils::concatenate(constructHeader_(*quantMethod), "\t"));
 
-    for (ConsensusMap::ConstIterator cm_iter = cm.begin();
-         cm_iter != cm.end();
-         ++cm_iter)
+    for (const ConsensusFeature& cf : cm)
     {
-      const ConsensusFeature& cFeature = *cm_iter;
+      const ConsensusFeature& cFeature = cf;
```

Comment:
minor: there are still some of these unneeded variables around ...

Reply:
Fixed.